### ISSUE TITLE
Make blog posts unique

### ISSUE THREAD

**Issue by Sjael** [MeldGames/meld-website#2]
# Problem
Walls of text are everywhere on the internet, and people filter out most things. Blog posts are inherently text heavy, and are typically given the benefit of the doubt since usually people are googling for that specific topic (which your post is probably about). 

Many blogs fall victim to either:

1. Fully giving into the 'wall of text' and making readers need to visualize everything
2. Going full cringe and injecting a gif or meme every 3 lines, ruining pacing.

Ideally you would strike a balance between 'information neutron star' and 'stringing along with dopamine bursts'. Elements that are subtle, perhaps interact-able would be best.

# Solutions (difficulty asc.)
 - Ambient animations
 - First load animations
 - Rich text highlighting to bring emphasis
   - Svg backgrounds
   - Animated svgs
 - Concept art + game assets
 - Graphs + charts
 - 3D viewer

Basically, things that have motion, whether naturally or through player input.
